Static.podspec

```diff
@@ -8,7 +8,8 @@ Pod::Spec.new do |spec|
   spec.source = { git: 'https://github.com/venmo/Static.git', tag: "v#{spec.version}" }
   spec.author = { 'Venmo' => 'ios@venmo.com', 'Sam Soffes' => 'sam@soff.es' }
 
+  spec.swift_versions = ['5.0', '5.1', '5.2']
   spec.platform = :ios, '8.0'
```

Comment:
@soffes thoughts on matching the iOS versions? (e.g. `.v9` in SPM vs. `8.0` here).